install/traf_checkset_env.sh

```diff
@@ -0,0 +1,243 @@
+fi
+
+# check the based command
+basecmds=(yum lsb_release awk cut uname)
+for basecmd in ${basecmds[@]}  
+do
+    basecmdpath=`which ${basecmd}`
+    if [ "x$?" != "x0" ]; then
+	case ${basecmd} in
+	    yum)
+		echo "ERROR: You must first install yum."
+		exit 1
+		;;
+	    lsb_release)
+		(${MY_YUM} install redhat-lsb) >>${LOGFILE}2>&1
+		if [ "x$?" = "x0" ]; then
+		    echo "ERROR: yum repo server has a error when run command [${MY_YUM} install redhat-lsb]."
+		    exit 1
+		fi
+		;;
+	    *)
+		echo "ERROR: command [${basecmd}] not exist. Make sure you have installed it, and have added it to the command path."
+		exit 1
+	esac
+    fi
+    echo "INFO: command ${basecmd} exist"
+done
+
+osname=`uname -o`
+cpuarch=`uname -m`
+echo "INFO: os is [${osname}], cpu architecture is [${cpuarch}]"
+
+osdistributor=`lsb_release -i | cut -d : -f 2-`
+case ${osdistributor} in
+    *RedHat*|*CentOS*)
+	echo "INFO: os distributor id is [${osdistributor}]"
+	;;
+    *)
+	echo "ERROR: The OS distribution is [${osdistributor}], but Trafodion only supports RedHat and CentOS presently."
+	exit 1
+esac
+
+osver=`lsb_release -r | cut -d : -f 2-`
+osvermajor=`echo $osver | cut -d . -f 1-1`
+osverminor=`echo $osver | cut -d . -f 2-2`
+case ${osvermajor} in
+    6|7)
+	echo "INFO: os version is [$osver]"
+	;;
+    *)
+	echo "ERROR: OS version is [$osver]. Only 6.x versions are presently supported."
+	exit 1
+esac
+
+# install all libs
+dependlibs=(
+	    epel-release alsa-lib-devel ant ant-nodeps apr-devel	\
+	    apr-util-devel boost-devel cmake device-mapper-multipath	\
+	    dhcp doxygen flex gcc-c++ gd git glibc-devel		\
+	    glibc-devel.i686 graphviz-perl gzip ${MY_JAVA_VER}-devel	\
+	    libX11-devel libXau-devel libaio-devel libcurl-devel	\
+	    libibcm.i686 libibumad-devel libibumad-devel.i686 libiodbc	\
+	    libiodbc-devel librdmacm-devel librdmacm-devel.i686		\
+	    libxml2-devel lua-devel lzo-minilzo net-snmp-devel		\
+	    net-snmp-perl openldap-clients openldap-devel		\
+	    openldap-devel.i686 openmotif openssl-devel			\
+	    openssl-devel.i686 perl-Config-IniFiles perl-Config-Tiny	\
+	    perl-DBD-SQLite perl-Expect perl-IO-Tty perl-Math-Calc-Units \
+	    perl-Params-Validate perl-Parse-RecDescent perl-TermReadKey	\
+	    perl-Time-HiRes protobuf-compiler protobuf-devel		\
+	    readline-devel rpm-build saslwrapper sqlite-devel unixODBC	\
+	    unixODBC-devel uuid-perl wget xerces-c-devel xinetd		\
+	    ncurses-devel)
+faillibs=()
+echo "INFO: install dependent library start"
+for dependlib in ${dependlibs[@]}
+do
+    (${MY_YUM} install ${dependlib}) >>${LOGFILE}2>&1
+    if [ "x$?" != "x0" ]; then
+	faillibs=("${faillibs[@]}" "${dependlib}")
+	echo "ERROR: install dependent library [${dependlib}] fail"
+    else
+	echo "INFO: install dependent library [${dependlib}] success"
+    fi
+done
+echo "INFO: install dependent library finish"
+
+# remove pdsh and qt-dev
+echo "INFO: remove pdsh and qt-dev trafodion scripts get confused."
```

Comment:
Maybe say "INFO: remove pdsh and qt-dev commands to avoid problems with trafodion scripts"